wherehows-frontend/app/utils/Search.java

```diff
@@ -34,7 +34,10 @@
           + "{\"prefix\": {\"name\": {\"value\": \"$VALUE\", \"boost\": 32}}}, "
           + "{\"match\": {\"name\": {\"query\": \"$VALUE\", \"boost\": 48}}}, "
           + "{\"match\": {\"name\": {\"query\": \"$VALUE\", \"type\": \"phrase\", \"boost\": 64}}}, "
-          + "{\"wildcard\": {\"urn\": {\"value\": \"*$VALUE*\", \"boost\": 8}}}, "
+          + "{\"wildcard\": {\"urn\": {\"value\": \"*$VALUE*\", \"boost\": 16}}}, "
```

Comment:
We should consider refactoring this (in a separate PR) using http://docs.oracle.com/javaee/7/api/javax/json/JsonObjectBuilder.html instead of hand crafting JSON string like this.